frontend/src/types/AssessmentTypes.ts

```diff
@@ -1,10 +1,46 @@
+import { Grade } from "../APIClients/types/UserClientTypes";
+
 export type StatusProperty = "DRAFT" | "PUBLISHED" | "ARCHIVED" | "DELETED";
 
+export enum TestTypes {
+  Beginning = "BEGINNING",
+  End = "END",
+}
+
 export type AssessmentTypes = {
   status: StatusProperty;
   name: string;
-  grade: string;
-  type: "BEGINNING" | "END";
+  grade: Grade;
+  type: TestTypes;
   country: string;
   region: string;
 };
+
+export const gradeOptions = [
+  "Kindergarten",
+  "Grade 1",
+  "Grade 2",
+  "Grade 3",
+  "Grade 4",
+  "Grade 5",
+  "Grade 6",
+  "Grade 7",
+  "Grade 8",
+].map((value) => ({ value, label: value }));
+
+export const testTypeOptions = [
+  { value: "BEGINNING", label: "Beginning" },
+  { value: "END", label: "End" },
+];
+
+export const regionOptions = [
+  { value: "Ottawa", label: "Ottawa" },
+  { value: "California", label: "California" },
+  { value: "Ontario", label: "Ontario" },
+  { value: "Texas", label: "Texas" },
+];
+
+export const countryOptions = [
+  { value: "Canada", label: "Canada" },
+  { value: "USA", label: "USA" },
+];
```

Comment:
for test data, can we define it in the file that we are testing instead of the `types` folder? so that we when we replace it with the actual data, we don't forget to delete it

Reply:
Oh I got told earlier to move these to types lol I'll move them back